app/util/sanitize.js

```diff
@@ -0,0 +1,12 @@
+const regHeaders = (data) => {
```

Comment:
regHeaders --> validatereAccessControlRequestHeaders

Reply:
good unit test coverage needed, ideally with some real values acceptable and imaginative bad values